python/llm/src/bigdl/llm/transformers/low_bit_linear.py

```diff
@@ -440,7 +440,8 @@ def forward(self, x: torch.Tensor):
                 # current workaround to reduce first token latency of fp32 input
                 # sometimes fp16 cause nan and training instability
                 # disable the conversion when training
-                if self.conver_to_half and x_2d.shape[0] > 1 and x_2d.dtype == torch.float32:
+                if self.conver_to_half and x_2d.shape[0] > 1 and x_2d.dtype == torch.float32 and \
```

Comment:
`x_2d.shape[0] > 1` no longer need?

Reply:
still needed. if we remove it, then other qtype such as `SYM_FP4` with `x_2d.shape[0] = 1` will use fp16, but all `x_2d.shape[0]=1` should use fp32